lib/irb/init.rb

```diff
@@ -395,33 +395,41 @@ def IRB.parse_opts(argv: ::ARGV)
   # Run the config file
   def IRB.run_config
     if @CONF[:RC]
-      begin
-        file = rc_file
+      rc_files.each do |rc|
         # Because rc_file always returns `HOME/.irbrc` even if no rc file is present, we can't warn users about missing rc files.
         # Otherwise, it'd be very noisy.
-        load file if File.exist?(file)
+        load rc if File.exist?(rc)
       rescue StandardError, ScriptError => e
-        warn "Error loading RC file '#{file}':\n#{e.full_message(highlight: false)}"
+        warn "Error loading RC file '#{rc}':\n#{e.full_message(highlight: false)}"
```

Comment:
With this warning, we need to wrap `rc_file` tests with nil verbose to avoid creating noise in test output. Like:

```rb
    def test_rc_file
      verbose, $VERBOSE = $VERBOSE, nil
      tmpdir = @tmpdir
      Dir.chdir(tmpdir) do
        ENV["XDG_CONFIG_HOME"] = "#{tmpdir}/xdg"
        IRB.conf[:RC_NAME_GENERATOR] = nil
        assert_equal(tmpdir+"/.irb#{IRB::IRBRC_EXT}", IRB.rc_file)
        assert_equal(tmpdir+"/.irb_history", IRB.rc_file("_history"))
        assert_file.not_exist?(tmpdir+"/xdg")
        IRB.conf[:RC_NAME_GENERATOR] = nil
        FileUtils.touch(tmpdir+"/.irb#{IRB::IRBRC_EXT}")
        assert_equal(tmpdir+"/.irb#{IRB::IRBRC_EXT}", IRB.rc_file)
        assert_equal(tmpdir+"/.irb_history", IRB.rc_file("_history"))
        assert_file.not_exist?(tmpdir+"/xdg")
      end
    ensure
      $VERBOSE = verbose
    end
```